test/new-e2e/tests/agent-platform/ddot/ddot_install_test.go

```diff
@@ -114,9 +126,39 @@ func (is *ddotInstallSuite) TestDDOTInstall() {
 		require.Equal(is.T(), *platform, "suse", "NonSupportedPlatformError : %s isn't supported !", *platform)
 		is.ddotSuseTest(VMclient)
 	}
+	is.ConfigureAndRunAgentService(VMclient)
 	is.CheckDDOTInstallation(VMclient)
 }
 
+func (is *ddotInstallSuite) ConfigureAndRunAgentService(VMclient *common.TestClient) {
+	is.T().Run("add config file", func(t *testing.T) {
+		ExecuteWithoutError(t, VMclient, "sudo sh -c \"sed 's/api_key:.*/api_key: XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX/' /etc/datadog-agent/datadog.yaml.example > /etc/datadog-agent/datadog.yaml\"")
+		ExecuteWithoutError(t, VMclient, "sudo sh -c \"printf 'otelcollector:\\n  enabled: true\\n' >> /etc/datadog-agent/datadog.yaml\"")
+		ExecuteWithoutError(t, VMclient, "sudo sh -c \"sed -e 's/\\${env:DD_API_KEY}/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX/' -e 's/\\${env:DD_SITE}/datadoghq.com/' /etc/datadog-agent/otel-config.yaml.example > /etc/datadog-agent/otel-config.yaml\"")
+		ExecuteWithoutError(t, VMclient, "sudo sh -c \"chown dd-agent:dd-agent /etc/datadog-agent/datadog.yaml && chmod 640 /etc/datadog-agent/datadog.yaml\"")
+		ExecuteWithoutError(t, VMclient, "sudo sh -c \"chown dd-agent:dd-agent /etc/datadog-agent/otel-config.yaml && chmod 640 /etc/datadog-agent/otel-config.yaml\"")
+		if (*platform == "ubuntu" && is.osVersion == 14.04) || (*platform == "centos" && is.osVersion == 6.10) {
+			ExecuteWithoutError(t, VMclient, "sudo initctl start datadog-agent")
+		} else {
+			ExecuteWithoutError(t, VMclient, "sudo systemctl restart datadog-agent.service")
+		}
+	})
+
+	is.T().Run("check ddot systemd units are running", func(t *testing.T) {
+		ddotUnit := is.host.State().Units["datadog-agent-ddot.service"]
+		require.NotNil(t, ddotUnit)
+		require.Equal(t, "datadog-agent-ddot.service", ddotUnit.Name)
+		require.Equal(t, "active", ddotUnit.Active)
+		require.Equal(t, host.Loaded, ddotUnit.LoadState)
+		require.Equal(t, host.Running, ddotUnit.SubState)
+
+		is.T().Log("--- ConfigureAndRunAgentService ---")
+		for s, u := range is.host.State().Units {
+			is.T().Log(s, u.Name, u.Enabled, u.Active, u.LoadState, u.SubState)
+		}
+	})
+}
```

Comment:
These seem to add a lot of [noise to test output](https://gitlab.ddbuild.io/DataDog/datadog-agent/-/jobs/1033028557#L210). I can see their use while debugging, but perhaps we no longer need it or we can at least filter for units that may be relevant to the test?